app/loader.h

```diff
@@ -40,6 +40,8 @@ class Loader: public QObject
     bool isRecording() const { return mRecording; }
     void setRecording( bool isRecording );
 
+    QString loadingFile() const;
```

Comment:
maybe rename to `loadingProjectProgressFilePath()` and add  some nice comment so no one confuses it with the path to the QGIS project file that loads....